infra/terraform/environments/dev/main.tf

```diff
@@ -7,7 +7,8 @@ terraform {
 }
 
 provider "aws" {
-  region = "${var.region}"
+  region  = "${var.region}"
+  version = ">= v1.25.0"
```

Comment:
Bumped AWS provider version to use [`aws_efs_file_system.dns_name`](https://www.terraform.io/docs/providers/aws/d/efs_file_system.html) attribute.